app/views/trips/index.html.erb

```diff
@@ -0,0 +1,2 @@
+<h1>Trips#index</h1>
```

Comment:
We should remove this template file since we don't have a `Trips#index` action at all.